.github/workflows/branch-build.yml

```diff
@@ -46,6 +46,8 @@ jobs:
         version: latest
     - name: Run GoReleaser
       run: make release-snapshot
+      env:
+        ANALYTICS_API_KEY: ${{ secrets.ANALYTICS_API_KEY }}
```

Comment:
IMO not, BTW should we do that on main? are we able to easily distinguish it from other real user installations?

Reply:
IMO not, BTW should we do that on main? are we able to easily distinguish it from other real user installations?

Reply:
Theoretically, users can use image built from main as well 🤷 I disabled analytics collection from CI pipeline for `main`, but I'd rather keep it for development builds on `main`.

Maybe let's keep it for now, and remove that if it will be really bloating the data. WDYT?

Reply:
discussed f2f, I meant enabled reporting, with API key, completely makes sens 👍